pkg/cmd/server/server.go

```diff
@@ -951,6 +952,8 @@ func (s *server) runControllers(defaultVolumeSnapshotLocations map[string]string
 		s.logger.Fatal(err, "fail to get controller-runtime informer from manager for PVR")
 	}
 
+	hooktracker := hook.NewMultiHookTracker()
```

Comment:
This is trivial but I'd like to suggest we rename the instance of multi-hook-tracker in the restore flow to distinguish  from the instance in backup flow

Reply:
Fixed.